From: Simon McVittie <smcv@debian.org>
Date: Mon, 21 Mar 2022 19:06:29 +0000
Subject: tests: Skip comparison of tzdata version number

The test expects the timezone database version number to be reported as
2021a3, but it's actually reported as 2021a.
---
 js/src/tests/jstests.list | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/js/src/tests/jstests.list b/js/src/tests/jstests.list
index 8fc397a..d34ca9e 100644
--- a/js/src/tests/jstests.list
+++ b/js/src/tests/jstests.list
@@ -640,6 +640,9 @@ skip script non262/Date/time-zones-imported.js
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1755531
 skip-if(xulRuntime.debian.DEB_HOST_ARCH_CPU.match(/i386|s390x|arm64|ppc64|riscv64/)) script test262/built-ins/Date/UTC/fp-evaluation-order.js
 
+# This test fails with the embedded ICU version (only)
+skip script non262/Intl/DateTimeFormat/timeZone_version.js
+
 ###########################################################
 # Tests disabled due to issues in test262 importer script #
 ###########################################################
